A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/cameras.cfg

```diff
@@ -1431,6 +1431,47 @@ NodesToHide =""
 ClipMode ="Normal"
 BoundingBoxRadius =0.1
 
+[CAMERADEFINITION.35]
+Title ="EFB (flyPad)"
+Guid ="{7173B746-1B81-4964-BEB3-106E68F91551}"
+Description =""
+Origin ="Virtual Cockpit"
+MomentumEffect =1
+SnapPbhAdjust ="Swivel"
+SnapPbhReturn =0
+PanPbhAdjust ="Swivel"
+PanPbhReturn =0
+Track ="None"
+ShowAxis ="YES"
+AllowZoom =1
+InitialZoom =0.1
+SmoothZoomTime =2
+ZoomPanScalar =1
+ShowWeather =1
+XyzAdjust =1
+ShowLensFlare =0
+Category ="Cockpit"
+SubCategory ="Instrument"
+SubCategoryItem ="None"
+PitchPanRate =20
+HeadingPanRate =60
+InitialXyz = -0.045, 0.77, -2.274
+InitialPbh = -30, 0, -62
```

Comment:
Just checking the second number doesn't control the "bank"/roll?

Reply:
No, supposedly it does in FSX (Pbh = pitch, bank, heading), but in MSFS it seems to be added to the pitch instead..

Reply:
It won't work because it's not a drone camera. Roll works for showcase. It is strange.

Reply:
Thanks for having a look. It is rather strange, especially that it just adds roll to pitch instead of ignoring it.